Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                var name = GetString("What is your name: ");
+                var surname = GetString("What is your surname: ");
+                var age = GetInt("What is your age: ");
+
+
+                var weight = GetFloat("What is your weight in kilogram: ");
+                var height = GetFloat("What is your height in metres: ");
+                var bmi = GetBmi(weight, height);
+
+                Console.Write($"{name}  {surname}  is {age} years old, his weight is {weight} kg and his height is {height:F2} cm.");
+                Console.WriteLine($"His BMI is:  {bmi:N2}");
+                Console.WriteLine("");
+                if (i == 0)
+                {
+                    Console.WriteLine($"Now for another person {Environment.NewLine}  ");
+                }
+            }
+        }
+
+        public static float GetBmi(float weight, float height)
+        {
+            if (height <= 0 || weight <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+
+                if (height <= 0 && weight <= 0)
+                {
+                    Console.WriteLine($"Weight cannot be equal or less than zero, but was 0.{Environment.NewLine}Height cannot be less than zero, but was 0.");
```

Comment:
You are saying here that "Weight cannot be equal or less than zero, but was 0." but it wasn't necessarily 0.  Should it be "Weight cannot be equal or less than zero, but was {weight}."

Reply:
is changed in my last commit